app/src/main/java/org/luxoft/sdl_core/SdlLauncherService.java

```diff
@@ -0,0 +1,107 @@
+    public final static String ON_SDL_SERVICE_STARTED = "ON_SDL_SERVICE_STARTED";
+
+    private static final String APP_ID = "sdl_service";
+
+    private static final int FOREGROUND_SERVICE_ID = 123;
+
+    private boolean is_first_load_ = true;
+    private Thread sdl_thread_ = null;
+
+    private native static void StartSDLNative();
+    private native static void StopSDLNative();
+
+    @Override
+    public IBinder onBind(Intent intent) {
+        return null;
+    }
+
+    @Override
+    public void onCreate() {
+        super.onCreate();
+        enterForeground();
+    }
+
+    @Override
+    public int onStartCommand(Intent intent, int flags, int startId) {
+
+        if (is_first_load_) {
+            try {
+                System.loadLibrary("c++_shared");
+                System.loadLibrary("emhashmap");
+                System.loadLibrary("bson");
+                System.loadLibrary("boost_system");
+                System.loadLibrary("boost_regex");
+                System.loadLibrary("boost_thread");
+                System.loadLibrary("boost_date_time");
+                System.loadLibrary("boost_filesystem");
+                System.loadLibrary("smartDeviceLinkCore");
+                is_first_load_ = false;
+            } catch (UnsatisfiedLinkError e) {
+                showToastMessage("Failed to load the library: " + e.getMessage());
+                return super.onStartCommand(intent, flags, startId);
+            }
+        }
+
+        sdl_thread_ = new Thread() {
+            @Override
+            public void run() {
+                StartSDLNative();
+            }
+        };
+        sdl_thread_.start();
+
+        final Intent start_intent = new Intent(ON_SDL_SERVICE_STARTED);
+        getApplicationContext().sendBroadcast(start_intent);
+
+        return super.onStartCommand(intent, flags, startId);
+    }
+    @Override
+    public void onDestroy() {
+
+        StopSDLNative();
+
+        final Intent intent = new Intent(ON_SDL_SERVICE_STOPPED);
+        getApplicationContext().sendBroadcast(intent);
+
+        super.onDestroy();
+    }
+
+    private void showToastMessage(String message) {
+        Context context = getApplicationContext();
+        int duration = Toast.LENGTH_LONG;
+
+        Toast toast = Toast.makeText(context, message, duration);
+        toast.show();
+    }
+
+    @SuppressLint("NewApi")
+    public void enterForeground() {
+        NotificationChannel channel = new NotificationChannel(APP_ID, "SdlService", NotificationManager.IMPORTANCE_DEFAULT);
+        NotificationManager notificationManager = (NotificationManager) getSystemService(Context.NOTIFICATION_SERVICE);
+        if (notificationManager != null) {
+            notificationManager.createNotificationChannel(channel);
+            Notification serviceNotification = new Notification.Builder(this, channel.getId())
+                    .setContentTitle("SDL")
```

Comment:
And this too
```suggestion
                    .setContentTitle(SDL_CONTENT_TITLE)
```

Reply:
fixed in f874c10935f6b28d6931e867ca35d2992f8b0469